packages/docusaurus-theme-classic/src/theme/DocItem/index.js

```diff
@@ -37,7 +37,7 @@ function Headings({headings, isChild}) {
       {headings.map(heading => (
         <li key={heading.id}>
           <a href={`#${heading.id}`} className={LINK_CLASS_NAME}>
-            {heading.value}
+            <div dangerouslySetInnerHTML={{__html: heading.value}} />
```

Comment:
This adds an extra `<div>` element. Why don't do `dangerouslySetInnerHTML` on the `<a>`?

Reply:
You are right. I am currently away so will update after that then